lib/ansible/modules/windows/win_find.ps1

```diff
@@ -331,7 +361,7 @@ foreach ($path in $paths_to_check) {
         $file = Get-Item -Force -Path $path
         $info = Get-FileStat -file $file
     } catch {
-        Add-Warning -obj $result -message "win_find failed to check some files, these files were ignored and will not be part of the result output"
+        Add-Warning -obj $result -message "win_find failed to check some files, these files were ignored and will not be part of the result output: $_"
```

Comment:
Maybe also change the part `failed to check some files` to `failed to check '$path'` so the user knows what file it actually was.

Reply:
Looks good, I think having a warning for each invocation is fine with me. Tells the user that they couldn't follow the link and why.